Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:18.04
+
+WORKDIR /app
+
+COPY ./output/goldennum /app
```

Comment:
We may copy entire ./output/* to /app, since templates and statics are not built into goldennum binary.